### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by mohametalmeari**
- [ ] Create animal table 
- [ ] Insert data into the table
- [ ] Select data from the table
image
image

### CODE REVIEW

**Comment by mohametalmeari** [comment]
Thank you for the review, but I already used the same template except I deleted unnecessary lines such as the lines before the table of contents because they included instructions for me. I also removed the testing section as there is no testing in this project, along with a few other unnecessary lines

**Comment by Olamarx** [comment]
> Thank you for the review, but I already used the same template except I deleted unnecessary lines such as the lines before the table of contents because they included instructions for me. I also removed the testing section as there is no testing in this project, along with a few other unnecessary lines

I understand but the line before the table is not unnecessary because it points to the file with some messages.

File: README.md [hunk 1]
```diff
@@ -248,44 +110,23 @@ Feel free to check the [issues page](../../issues/).
...
+I would like to thank the Microverse Team for their invaluable support in enhancing my backend skills.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
```
Comment (AlphaNtihinduka) [inline README.md:129]: - [ ] I have noticed that your MIT license link is broken. Kindly ensure that you add the `MIT`. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @mohametalmeari 🙋‍♂️
Good job creating the animal table . You have done incredibly well. 👏  

TO HIGHLIGHT:
- Tables structure changes saved into schema.sql
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
_- Kindly consider the inline review comments ☺️._

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by prolajumokeoni — APPROVED** [review]
Hi @mohametalmeari 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - All changes requested have been implemented 🥇  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mohametalmeari** [comment]
Thank you for the review

**Pull request merged**